packages/snap/src/helpers/proof-helpers.ts

```diff
@@ -528,6 +531,8 @@ export const prepareCircuitArrayValues = (
 };
 
 export const getNodeAuxValue = (p: Proof | undefined): NodeAuxValue => {
+  const nodeAux = isEmpty(p?.nodeAux) ? get(p, 'node_aux') : p?.nodeAux;
```

Comment:
Simplify it a bit, mb?
```suggestion
  const nodeAux = p.nodeAux ?? get(p, 'node_aux');
```

Reply:
Not sure what will come to, it could be `{}`, or `undefined`